flask_praetorian/base.py

```diff
@@ -484,3 +550,111 @@ def pack_header_for_user(
             **custom_claims
         )
         return {self.header_name: self.header_type + ' ' + token}
+
+    def send_registration_email(self, user=None, template=None,
+                                subject=None,
+                                **kwargs):
+        """
+        Sends a registration email to a new user, containing a time expiring
+            token usable for validation.
+        :param: user:                     The user object to tie claim to
+                                          (username, id, email, etc)
+        :param: subject:                  The registration email subject
+                                          override.
+        :param: override_access_lifepsan: Anything used in `encode_jwt_token`,
+                                          but all we really care about is
+                                          `override_access_token`.  This
+                                          overrides the instance's defined
+                                          lifespan, which is used as the
+                                          default.  The token will no longer
+                                          be valid after this time and
+                                          be rejected.
+        :param: template:                 Template file location for email.
+                                          If not provided, a stock entry is
+                                          used.
+        """
+
+        class Notification:
```

Comment:
I think this would be better as a nameduple

Reply:
submitted